utils/helpers.ts

```diff
@@ -184,3 +184,74 @@ export const cleanQuickstartName = (str: string): string =>
     .replace(/\s+/g, '-')
     .replace(/-+/, '-')
     .replace(/[^a-z0-9-]/g, '');
+
+/**
+ * Gets the unique base quickstart directory from a given file path.
+ * e.g. filePath: 'quickstarts/python/aiohttp/alerts/ApdexScore.yml' + targetChild: 'alerts' = 'python/aiohttp'.
+ * @param {String} filePath - Full file path of a file in a quickstart.
+ * @param {String} targetChild - Node in file path that should be preceded by a base quickstart directory.
+ * @return {String} Node in file path of the quickstart.
+ */
+const getQuickstartNode = (
+  filePath: string,
+  targetChild: string | undefined
+): string => {
+  const splitFilePath = filePath.split('/');
+
+  const baseQuickstartDirectoryIndex =
+    splitFilePath.findIndex((path) => path === targetChild) - 1;
+
+  let uniqueQuickstartDirectory = splitFilePath[baseQuickstartDirectoryIndex];
+  let indexCounter = baseQuickstartDirectoryIndex;
+
+  while (indexCounter > 1) {
+    uniqueQuickstartDirectory = splitFilePath[indexCounter - 1].concat(
+      `/${uniqueQuickstartDirectory}`
+    );
+    indexCounter--;
+  }
+  return uniqueQuickstartDirectory;
+};
+
+/**
+ * Identifies where in a given file path to look for a quickstart directory.
+ * @param {String} filePath - Full file path of a file in a quickstart.
+ * @return {String|undefined} Called function with arguments to determine the quickstart of a given file path.
```

Comment:
Same comment as above. Totally optional.
```suggestion
 * @param filePath - Full file path of a file in a quickstart.
 * @return Called function with arguments to determine the quickstart of a given file path.
```

Reply:
Done!